exports.go

```diff
@@ -60,8 +60,8 @@ type ServiceLatency struct {
 }
 
 func (sl *ServiceLatency) Validate(vr *ValidationResults) {
-	if sl.Sampling < 1 || sl.Sampling > 100 {
-		vr.AddError("sampling percentage needs to be between 1-100")
+	if sl.Sampling < 0 || sl.Sampling > 100 {
```

Comment:
Not sure this is correct. We do the same check in the server when parsing local config: https://github.com/nats-io/nats-server/blob/master/server/opts.go#L1932.
When updating an account, if the jwt.Export.Latency (a *jwt.ServiceLatency object) is not nil, then server tries to track for latency and will fail if sampling is < 1 or > 100.
So not sure what is the right thing to do here... @derekcollison wdyt?

Reply:
we had an informal discussion where we said that 0 would disable the check thus allowing the configuration to have the subject etc, but be turned off easily. Perhaps this is not correct but `nsc` does allow the zero.

Reply:
I think its correct, we should remove it all together to disable IMO.